packages/tupaia-web/src/features/Map/Map.tsx

```diff
@@ -145,7 +150,7 @@ export const Map = () => {
           zoom={zoom as LeafletMapProps['zoom']}
           shouldSnapToPosition
         >
-          <TileLayer tileSetUrl={activeTileSet.url} showAttribution />
+          <TileLayer tileSetUrl={activeTileSet.url} showAttribution={false} />
```

Comment:
Just checking, was this something that just got changed to be true? If so, will setting it to false undo that fix?

Reply:
It was but now I've upgraded the attribution to a custom one to match what was on tupaia before.